src/tictactoe/game_runner.clj

```diff
@@ -1,35 +1,18 @@
 (ns tictactoe.game-runner
-  (:require [clojure.string :as string]))
+  (:require [clojure.string :as string]
+            [clojure.math.numeric-tower :as math]))
 
-(def game-types
-  {"1" "Human vs. Human" "2" "Human vs. Easy Computer"})
+(def ^:dynamic sleep-time 5000)
 
-(def board-sizes
+(def ^:private game-types
+  {"1" "Human vs. Human" "2" "Human vs. Impossible Computer"})
+
+(def ^:private board-sizes
   {"1" "3x3" "2" "4x4"})
 
-(def markers 
+(def ^:private markers 
   (set (string/split "abcdefghijklmnopqrstuvwxyz" #"")))
 
-(defn ^:private welcome-message
-  []
-  "Welcome to Tic Tac Toe!\n")
-
-(defn ^:private game-type-message
-  []
-  "Pick a game type:")
-
-(defn ^:private board-type-message
-  []
-  "What size board would you like to play?")
-
-(defn ^:private player-marker-message
-  [player]
-  (str "Please pick " player "'s marker (A-Z)"))
-
-(defn ^:private game-choices-message
-  []
-  "You've chosen:")
-
 (defn ^:private print-message
```

Comment:
Your tests pass when I comment out the second `[message supplementing-info]` version of this function

Reply:
That's interesting - when I comment it out all my tests fail due to wrong number of arguments.

Reply:
Sorry. Poorly worded. I just commented out the implementation of that arity.

```
(defn ^:private print-message
  ([message]
   (println message))
  ([message supplementing-info]
   #_(println message "=>" supplementing-info)))
```